industrial_ci/src/tests/clang_format_check.sh

```diff
@@ -19,10 +19,10 @@ function prepare_clang_format_check() {
   true
 }
 
-function run_clang_format_check() {
-  local err=0
-  local path
-  ici_make_temp_dir path
+function install_clang_format_check() {
```

Comment:
This function never gets called..

Reply:
Indeed, thanks for pointing this out - I made a wrong assumption about the calling order. I've now simplified it. It works by e.g. setting `BEFORE_RUN_CLANG_FORMAT_CHECK="wget [my config] -O /tmp/clang_format_check/[project]/.clang-format"` along with `ADDITIONAL_DEBS=wget` in this case.